### PR TITLE
`FormView` - Add support for Combo Box inputs + general refactoring

### PR DESCRIPTION
**Pull request by dfeinzimer**
- Adds support for Combo Box inputs in `FormView`s
- Adds starter Combo Box test cases 1-3
- Includes general `FormView` refactoring:
  - Renames previously completed inputs suffix from `Entry` to `Input`
  - Declares a common `InputFooter` view
  - Renames `FormElementHeader` -> `InputHeader`
  - Unifies and generalizes some repeated localized strings and updates the generated strings file
  - Removes some unused things from FormViewTestView.swift
  - Declares a temporary `FormFieldElement.isRequired` property based on the required expression. This can be removed as soon as the API for this is integrated.

### CODE REVIEW

File: Sources/ArcGISToolkit/Components/Form/FormInputs/ComboBoxInput.swift [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+            .searchable(text: $filterPhrase, placement: .navigationBarDrawer, prompt: .filter)
+            .navigationTitle(element.label)
+            .navigationBarTitleDisplayMode(.inline)
+            .toolbar {
+                ToolbarItem(placement: .navigationBarTrailing) {
+                    Button {
+                        isPresented = false
+                    } label: {
+                        Text.done
+                            .bold()
```
Comment (mhdostal) [inline Sources/ArcGISToolkit/Components/Form/FormInputs/ComboBoxInput.swift:174]: Why are some of the `Text.done` instances `.semiBold()` and this one is `.bold()`?

File: Sources/ArcGISToolkit/Resources/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -85,6 +85,15 @@ network element for more than one trace starting point. */
 for a utility network trace. */
 "Element could not be identified." = "Element could not be identified.";
 
+/* Text indicating a field's exact number of required characters. */
```
Comment (mhdostal) [inline Sources/ArcGISToolkit/Resources/en.lproj/Localizable.strings:88]:
```suggestion
/* Text indicating the user should enter a field's exact number of required characters. */
```

File: Sources/ArcGISToolkit/Resources/en.lproj/Localizable.strings [hunk 3]
```diff
@@ -85,6 +85,15 @@ network element for more than one trace starting point. */
...
+"Enter %lld characters" = "Enter %lld characters";
+
+/* Text indicating a field's minimum and maximum number of allowed characters. */
```
Comment (mhdostal) [inline Sources/ArcGISToolkit/Resources/en.lproj/Localizable.strings:91]:
```suggestion
/* Text indicating the user should enter a number of characters between a field's minimum and maximum number of allowed characters. */
```

**Review by mhdostal — CHANGES_REQUESTED** [review]

File: Sources/ArcGISToolkit/Components/Form/FormInputs/ComboBoxInput.swift [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+        return codedValues
+            .filter { $0.name.localizedStandardContains(filterPhrase) }
+    }
+    
+    /// Creates a view for a combo box input.
+    /// - Parameters:
+    ///   - featureForm: The feature form containing the input.
+    ///   - element: The field's parent element.
+    ///   - input: The input configuration of the field.
+    init(featureForm: FeatureForm?, element: FieldFormElement, input: ComboBoxFormInput) {
```
[the remark this answers is not in the document]
Comment (dfeinzimer) [reply Sources/ArcGISToolkit/Components/Form/FormInputs/ComboBoxInput.swift:60]: Good call, I will note this down and follow up on it in a separate PR.

**Review by mhdostal — APPROVED** [review]

**Pull request merged**